src/olympia/core/languages.py

```diff
@@ -1,252 +1,245 @@
-    'th': {'english': 'Thai', 'native': '\u0e44\u0e17\u0e22'},
+    'ta': {'english': 'Tamil', 'native': 'தமிழ்'},
+    'ta-IN': {'english': 'Tamil (India)', 'native': 'தமிழ் (இந்தியா)'},
+    'ta-LK': {'english': 'Tamil (Sri Lanka)', 'native': 'தமிழ் (இலங்கை)'},
+    'te': {'english': 'Telugu', 'native': 'తెలుగు'},
+    'th': {'english': 'Thai', 'native': 'ไทย'},
     'tl': {'english': 'Tagalog', 'native': 'Tagalog'},
     'tn': {'english': 'Tswana', 'native': 'Setswana'},
+    'tr': {'english': 'Turkish', 'native': 'Türkçe'},
     'ts': {'english': 'Tsonga', 'native': 'Xitsonga'},
-    'tsz': {'english': 'Pur\xe9pecha', 'native': 'Pur\xe9pecha'},
-    'tt-RU': {'english': 'Tatar', 'native': 'Tatar\xe7a'},
-    'ur': {'english': 'Urdu', 'native': '\u0627\u064f\u0631\u062f\u0648'},
-    'uz': {'english': 'Uzbek', 'native': 'O\u02bbzbek tili'},
-    've': {'english': 'Venda', 'native': 'Tshiven\u1e13a'},
+    'tsz': {'english': 'Purépecha', 'native': 'Purépecha'},
+    'tt-RU': {'english': 'Tatar', 'native': 'Tatarça'},
+    'uk': {'english': 'Ukrainian', 'native': 'Українська'},
+    'ur': {'english': 'Urdu', 'native': 'اُردو'},
+    'uz': {'english': 'Uzbek', 'native': 'Oʻzbek tili'},
+    've': {'english': 'Venda', 'native': 'Tshivenḓa'},
+    'vi': {'english': 'Vietnamese', 'native': 'Tiếng Việt'},
     'wo': {'english': 'Wolof', 'native': 'Wolof'},
-    'x-testing': {
-        'english': 'Testing',
-        'native': '\u0166\u1e17\u015f\u0167\u012b\u019e\u0260',
-    },
+    'x-testing': {'english': 'Testing', 'native': 'Ŧḗşŧīƞɠ'},
     'xh': {'english': 'Xhosa', 'native': 'isiXhosa'},
-    'yo': {'english': 'Yoruba', 'native': 'Yor\xf9b\xe1'},
+    'yo': {'english': 'Yoruba', 'native': 'Yorùbá'},
+    'zh-CN': {'english': 'Chinese (Simplified)', 'native': '中文 (简体)'},
+    'zh-TW': {'english': 'Chinese (Traditional)', 'native': '正體中文 (繁體)'},
     'zu': {'english': 'Zulu', 'native': 'isiZulu'},
 }
 
 
-ALL_LANGUAGES = {**AMO_LANGUAGES, **ADDITIONAL_PRODUCT_LANGUAGES}
+# these are used by prod and stage settings
+PROD_LANGUAGES = [
+    'cs',
+    'de',
+    'dsb',
+    'el',
+    'en-CA',
+    'en-GB',
+    'en-US',
+    'es-AR',
+    'es-CL',
+    'es-ES',
+    'es-MX',
+    'fi',
+    'fr',
+    'fur',
+    'fy-NL',
+    'he',
+    'hr',
+    'hsb',
+    'hu',
+    'ia',
+    'id',
+    'it',
+    'ja',
+    'ka',
+    'kab',
+    'ko',
+    'nb-NO',
+    'nl',
+    'nn-NO',
+    'pl',
+    'pt-BR',
+    'pt-PT',
+    'ru',
+    'sk',
+    'sl',
+    'sq',
+    'sv-SE',
+    'tr',
+    'uk',
+    'vi',
+    'zh-CN',
+    'zh-TW',
+]
+
+# Languages we can support, but have to fake support for Babel.
+# See olympia.amo.utils.get_locale_from_lang
+LANGUAGES_NOT_IN_BABEL = [
+    'ach',
+    'azz',
+    'cak',
+    'csb',
+    'fj-FJ',
+    'son',
+    'tl',
+    'tsz',
+]
+
+# Languages we've historically supported for statistics, etc, but we can't use because
+# they have a format we don't support in urls
+UNSUPPORTED_LANGUAGES = [
```

Comment:
I think we should probably keep them around just to be aware of them, but prevent them from being ever accidentally enabled with a simple test proving they are not in `PROD_LANGUAGES`.

Reply:
added in https://github.com/mozilla/addons-server/pull/23747/commits/80d12df4c75deae15e1ea50d822053319e292ed1